rc/windowing/kitty.kak

```diff
@@ -54,13 +46,12 @@ If no client is passed then the current one is used' \
         if [ $# -eq 1 ]; then
             printf "evaluate-commands -client '%s' focus" "$1"
         else
-            if [ -z "$kak_client_env_KITTY_LISTEN_ON" ]; then
-                kitty @ focus-tab --no-response --match=id:$kak_client_env_KITTY_WINDOW_ID
-                kitty @ focus-window --no-response --match=id:$kak_client_env_KITTY_WINDOW_ID
-            else
-                kitty @ --to "$kak_client_env_KITTY_LISTEN_ON" focus-tab --no-response --match=id:$kak_client_env_KITTY_WINDOW_ID
-                kitty @ --to "$kak_client_env_KITTY_LISTEN_ON" focus-window --no-response --match=id:$kak_client_env_KITTY_WINDOW_ID
+            listen=""
+            if [ -n "$kak_client_env_KITTY_LISTEN_ON" ]; then
+                listen="--to=$kak_client_env_KITTY_LISTEN_ON"
             fi
+
+            kitty @ "$listen" focus-window --no-response --match=id:"$kak_client_env_KITTY_WINDOW_ID"
```

Comment:
I think `"$listen"` should not be quoted here, as we want an empty listen not-to be passed as an empty parameter

Reply:
yes!